### ISSUE TITLE
[YouTube Downloader] ...

### ISSUE THREAD

**Issue by GaferFx** [raycast/extensions#8337]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

#### Error:
```
SyntaxError: Invalid or unexpected token

Ama=function(a,b,c,d){var e=null

### Steps To Reproduce

under normal usage

### Current Behaviour

_No response_

### Expected Behaviour

_No response_

**Comment by pernielsentikaer** [comment]
What URL did you test with?

**Comment by GaferFx** [comment]
> 

https://youtu.be/nOOrK9KBQBY?si=gJONyGo89jGH-OOC

**Comment by darmiel** [comment]
See #8296

**Comment by darmiel** [comment]
@GaferFx Can you check if it's working again by now?

**Comment by vimtor** [comment]
This got "fixed" in #9007.

@raycastbot close this issue
